src/riakc_ts.erl

```diff
@@ -186,3 +188,12 @@ server_call(Pid, Message) ->
     gen_server:call(Pid,
                     {req, Message, riakc_pb_socket:default_timeout(timeseries)},
                     infinity).
+-spec convert_to_binary(Value :: binary() | list()) -> binary().
```

Comment:
@lukebakken I thought the builtin `iolist_to_binary` was more elegant, which is why I didn't push my other PR.  Why switch to my home-brewed function?